lib/tasks/import-graduation.rake

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+files = [
+          './app/assets/data/graduation/2013-14-5YearGraduation-all.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-american_indian_or_alaskan_native.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-asian.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-asian_pacific_islander.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-black_or_african_american.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-female.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-foster_care.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-free_or_reduced_price_lunch.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-hispanic_or_latino.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-homeless.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-male.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-section_504.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-special_education.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-title_i_migrant.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-transitional_bilingual.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-two_or_more_races.csv',
+          './app/assets/data/graduation/2013-14-5YearGraduation-white.csv',
+        ]
+
+namespace :db do
+  task import_graduation_data: :environment do
+    files.each do |file|
+      CSV.foreach(file, headers: true) do |row|
+        identifier_name = file.chomp('.csv').split('-')[-1].split('_').join(" ")
```

Comment:
I dislike how dependent this is on the file-naming structure - seems too prone to user error. Ideas? It's not a row in the table